Tools/jhbuild/jhbuild-wrapper

```diff
@@ -24,7 +24,7 @@ import shlex
 import subprocess
 import sys
 
-jhbuild_revision = '048920f64d82779eef88158455bfd1d160a3be1a'
+jhbuild_revision = 'master'
```

Comment:
Please don't set master.

That introduces the risk of random regressions because we can't be sure what is pulled from upstream is the same always.
Is ok to update to the last current master, but use the git hash rather than the master key.